cpp/src/io/text/multibyte_split.cu

```diff
@@ -717,7 +719,7 @@ std::unique_ptr<cudf::column> multibyte_split(cudf::io::text::data_chunk_source
     // while that is running, determine how many offsets we output (synchronizes)
     auto next_tile_offset =
       tile_offsets.get_inclusive_prefix(base_tile_idx + tiles_in_launch - 1, scan_stream);
-    offset_storage.advance_output(next_tile_offset.offset() - offset_storage.size());
+    offset_storage.advance_output(next_tile_offset.offset() - offset_storage.size(), stream);
```

Comment:
I find it a bit surprising that one member call uses `scan_stream` and the other one uses `stream`. Do you see any harm in using `scan_stream` everywhere? If you're worried about the stream pools "leaking", we could introduce another scope between fork_stream and join_stream?